product.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/furnitureDatabaseConnector.php';
+require_once 'src/Entities/ProductEntity.php';
+
+$db = furnitureDatabaseConnector::connect();
+$product = [];
```

Comment:
```suggestion
$product = false;
```

Type flip-flopping causes bugs.